<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>쓰레기통</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image.png</normaloff>image.png</iconset>
  </property>
  <widget class="QPushButton" name="main_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>560</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>회원 인증</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>260</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>신고</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>260</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
